### DISCUSSION TITLE
Superscript text

### DISCUSSION THREAD

**Discussion by pipdata** [scalar/scalar#1678] [Q&A] [answered]
I have tried to add some text as superscript, in plain html as `<sup>Text</sup>`, but it's not rendered correctly in the documentation page.
Is there an alternative?
Thanks.

**Comment by cameronrohani** [comment]
Will get team to review and get in tomorrow :) 
https://github.com/scalar/scalar/pull/1679

**Comment by pipdata** [comment]
Thank you @cameronrohani 
I can get something with raw Unicode, but it's not quite the same. Compare the html `A<sup>plus</sup>` with `A&#7510;&#737;&#7512;&#x02E2`
That's why I am trying to write `<sup>...</sup>` in the `.yml` file.

**Comment by pipdata** [comment, marked as answer]
I believe this was addressed in https://github.com/scalar/scalar/pull/1679 and made available in v 1.22.8
Thanks!
